libraries/Wire/src/utility/xmc_i2c_conf.c

```diff
@@ -197,40 +197,6 @@ XMC_I2C_t XMC_I2C_0 =
     .protocol_irq_service_request       	  = 5
 };
 
-#elif defined(XMC1400_Arduino_Kit) 
```

Comment:
Same as [xmc_spi_conf.c](https://github.com/Infineon/XMC-for-Arduino/pull/195/files#diff-f0a5b6e4a58530712f1cbce9463eed9daedae46d0f0acb1f8fde3641bd66ed4c)